benchmark/paddle/CARLA_SAC/env_utils.py

```diff
@@ -0,0 +1,125 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import parl
+import carla
+import gym
+import gym_carla
+import numpy as np
+from parl.utils import logger, tensorboard
+from parl.env.continuous_wrappers import ActionMappingWrapper
+
+
+class ParallelEnv(object):
+    def __init__(self, env_name, xparl_addr, train_envs_params):
+        parl.connect(xparl_addr)
```

Comment:
Users are encouraged to connect to the server at the beginning.
Please put this line at the main function.